TARS/src/mcu_main/telemetry.cpp

```diff
@@ -159,9 +159,26 @@ void Telemetry::transmit() {
     digitalWrite(LED_BLUE, blue_state);
     blue_state = !blue_state;
 
-    TelemetryPacket packet = makePacket(dataLogger.read());
+    Packets packet;
+
+    auto rocket_state = dataLogger.read().rocketState_data.rocketStates[0];
```

Comment:
Let's avoid using auto lol

Reply:
Also I might be wrong, but I think you can also use the `getActiveFSM()` function to get the state, it would make this line a little more clear. See how it's obtained in `Controller::ActiveControl_ON()` in `ActiveControl.cpp`